kong/runloop/wasm.lua

```diff
@@ -402,6 +402,8 @@ local function rebuild_state(db, version, old_state)
 
       for _, filter in ipairs(chain.filters) do
         if filter.enabled then
+          _M.filters_by_name[filter.name].config = cjson_decode(filter.config) or filter.config
```

Comment:
There may exist more than one filter by the same name (and likely with differing configuration).

Reply:
Indeed. I changed the code to use the filter metadata instead of instance configuration.